### PR TITLE
feat: check table status when CLI login

### LINKED ISSUE #3505: CLI check all table status when login
We'll get
image

### PR DESCRIPTION
**Pull request by vagetablechicken**
closes #3505 
print unhealthy table msg and give a hint to console

### CODE REVIEW

File: src/cmd/sql_cmd.h [hunk 1]
```diff
@@ -141,14 +141,53 @@ std::string ExecFetch(const std::string& sql) {
...
+            if (pu != "0") {
+                is_broken = true;
+                msg.append(", unalive partition: ").append(pu);
+            }
+            if (!warnings.empty()) {
+                is_broken = true;
+                msg.append(", warning preview: ").append(warnings.substr(0, 100));
+            }
+            if (is_broken) {
+                LOG(WARNING) << msg;
```
Comment (vagetablechicken) [inline src/cmd/sql_cmd.h]: Is it better to print console?

File: src/cmd/sql_cmd.h [hunk 2]
```diff
@@ -141,14 +141,53 @@ std::string ExecFetch(const std::string& sql) {
...
+        LOG(ERROR) << "fail to get all table status, " << status.ToString();
+    }
 }
 
 // cluster mode if zk_cluster is not empty, otherwise standalone mode
 void Shell() {
     DCHECK(cs);
     DCHECK(sr);
+    // before all, check all table status
+    CheckAllTableStatus();
```
Comment (vagetablechicken) [inline src/cmd/sql_cmd.h]: add hint, openmldb_tool inspect can get full and clear report

Comment (vagetablechicken) [reply src/cmd/sql_cmd.h]: image

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**